### PR TITLE
lz: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Again a command with no options resulting in a short tldr page.

### CODE REVIEW

File: pages/linux/lz.md
Comment (reinhart1010) [inline pages/linux/lz.md:3]:
```suggestion
> List all files inside a '.tar.gz' compressed archive.
```

File: pages/linux/lz.md
Comment (reinhart1010) [inline pages/linux/lz.md:6]:
```suggestion
- List all files inside a compressed archive:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks for contributing! I'd like to suggest the following changes to simplify the command explanations.

**Review by reinhart1010 — APPROVED** [review]
Thanks! But we will need to wait for another maintainer to approve this before doing the merge.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**
